charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -77,6 +77,10 @@ spec:
 {{- else }}
             - name: MANAGEMENT_ENDPOINT
               value: "{{- .Values.orbit.endpoint -}}"
+            - name: MANAGEMENT_MODE
```

Comment:
Likely these need to be added to the backbeat chart as well but otherwise LGTM

Reply:
@ssalaues you are right about `PUSH_ENDPOINT`. `MANAGEMENT_MODE` is not used by backbeat (and in the near future will not be used by cloudserver either)

Reply:
I believe the recent changes to backbeat has it get the configurations from mongo, so it doesn't require the management/push endpoints.